Lib/fontTools/feaLib/ast.py

```diff
@@ -836,23 +982,31 @@ def asFea(self, indent=""):
 
 
 class LookupReferenceStatement(Statement):
+    """Represents a ``lookup ...;`` statement to include a lookup in a feature.
+
+    The ``lookup`` should be a :class:`LookupBlock` object."""
     def __init__(self, lookup, location=None):
         Statement.__init__(self, location)
         self.location, self.lookup = (location, lookup)
 
     def build(self, builder):
+        """Calls the builder object's ``add_lookup_call`` callback."""
         builder.add_lookup_call(self.lookup.name)
 
     def asFea(self, indent=""):
         return "lookup {};".format(self.lookup.name)
 
 
 class MarkBasePosStatement(Statement):
+    """A mark-to-base positioning rule. The ``base`` must be a
+    `glyph-containing object`_. The ``marks`` should be a list of
```

Comment:
Mixing “must” and “should” implies a meaningful difference. Is the type of `marks` less important than the type of `base`?